Extension/src/LanguageServer/configurations.ts

```diff
@@ -739,29 +740,50 @@ export class CppProperties {
         const result: string[] = [];
         if (paths) {
             paths = this.resolveDefaults(paths, defaultValue);
-            paths.forEach(entry => {
-                const entries: string[] = util.resolveVariables(entry, env).split(util.envDelimiter).map(e => this.resolvePath(e, false)).filter(e => e);
-                result.push(...entries);
-            });
-        }
-        if (!glob) {
-            return result;
+            if (paths && !glob) {
```

Comment:
What's the check for paths on line 743 for? `this.resolveDefaults(paths, defaultValue)` always returns an array but length of array could be 0. Is it supposed to check if paths has items instead?

Reply:
That was to make sure it wasn't undefined but that is fixed through your earlier suggestion.